### PR TITLE
Better name for the DLQ override

### PR DESCRIPTION
**Pull request by andreasohlund**
See https://github.com/Particular/docs.particular.net/pull/2437#discussion_r101884863 for the reasoning

### CODE REVIEW

File: src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs [hunk 1]
```diff
@@ -62,11 +62,11 @@ public static InstanceMappingFileSettings InstanceMappingFile(this RoutingSettin
         }
 
         /// <summary>
-        /// Moves expired messages using the "time to reach queue" setting to the dead letter queue instead of discarding them.
+        /// Moves messages that has expired their TimeToBeReceived to the dead letter queue instead of discarding them.
```
Comment (danielmarbach) [inline src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs]: Moves messages that have expired their TimeToBeReceived ?

Comment (andreasohlund) [inline src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs]: Good catch, fixed

File: src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs [hunk 2]
```diff
@@ -62,11 +62,11 @@ public static InstanceMappingFileSettings InstanceMappingFile(this RoutingSettin
         }
 
         /// <summary>
-        /// Moves expired messages using the "time to reach queue" setting to the dead letter queue instead of discarding them.
+        /// Moves messages that have expired their TimeToBeReceived to the dead letter queue instead of discarding them.
```
Comment (bording) [inline src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs]: How about "Moves messages that have **exceeded** their TimeToBeReceived" instead? Using "expired" in this context sounds awkward.

Comment (andreasohlund) [inline src/NServiceBus.Core/Transports/Msmq/MsmqConfigurationExtensions.cs]: :+1: fixed

**Pull request merged**